### ISSUE TITLE
Document Yarn version

### ISSUE THREAD

**Issue by eric-burel** [sysgears/apollo-universal-starter-kit#1108]
Related to #938 , we should update this part of the doc:

> 
> 2. Install the dependencies. Make sure that you use Yarn 1.0.0 or higher.
> 
> ```
> yarn
> ```
> 
> You can use NPM instead of Yarn to handle the starter kit dependencies and to run scripts. Throughout the Apollo
> Universal Starter Kit documentation, we'll always use Yarn.

Yarn 1.12.x won't work, so we could suggest to avoid it. Also I am not sure about saying you can use NPM instead of yarn. 
It worked for adding packages in specific app with `npm install`, however to upgrade a package (eg `extract-files` to version 5.0.0 for Storybook to work), I've had to run `yarn upgrade` from the root of the app, otherwise it wouldn't update the `yarn.lock`.
Basically that's a tripping point for people not used to yarn especially in a boilerplate with multiple environment, so I think it's worth it to give a bit more details about yarn usage in the Readme or in other part of the doc.

**Comment by vlasenko** [comment]
@eric-burel Thank you Eric, good point! Yes, we no longer support npm, because we use Yarn workspaces

**Comment by eric-burel** [comment]
Also some issue I have is that `core-js` gets installed and added to the lock with version 1.0.0 whenever I use `yarn add` in `packages/server`. There might be one package with bad dependencies but I can't tell which. I have to remove it from the `yarn.lock` afterward and rerun yarn.
